OCABundles/schema/bcgov-digital-trust/person-showcase/OCABundle.json

```diff
@@ -14,18 +14,8 @@
         "street_address": "Text"
       },
       "classification": "",
-      "digest": "ES3khN7HC1zZyr-DeQ6UbUVyhY_npOrxX_8NO_K4IWH8",
-      "flagged_attributes": [
```

Comment:
I wondering why the PII flag is being taken off of these attributes. Isn’t that exactly what they are?  I kind of get it for the Lawyer credential, but not for the person credential.

Or is it just because this is the “showcase” person credential?